ui.R

```diff
@@ -1,4 +1,9 @@
 
+variables <- c( "R1","R2","R1C","WO1","I")
+variables1 <- c("A1","A2","A1B","CM5","CM5B")
+variables2 <- c("CM1","CM2","CM4")
+variables3 <- c("I1","I9")
+variables4 <- c("RC1","RC4","RC9")
```

Comment:
Sound great! That will really help keep the tool more general so that it is easily applied to other agencies.